### DISCUSSION TITLE
How to embed the asciinema player in Wiki.js page

### DISCUSSION THREAD

**Discussion by lawrenceching** [requarks/wiki#6416] [Help / Questions]
Hi,

asciinema is a service to share terminal session.
It provides a simple html script and allow site owner to embed player in web page.

Follow its offical page(https://asciinema.org/docs/embedding), I can write a simplest html page and replay my terminal interaction.
```
<html>
	<body>
		<p>This is some text.</p>
		<script src="https://asciinema.org/a/14.js" id="asciicast-14" async></script>
		<p>This is some other text.</p>
	</body>
</html>
```

But the scritp tag didn't work in both Wiki.js Markdown page and HTML page.

Am I do it in a wrong way? Or it's a limitation from Wiki.js?

**Comment by NGPixel** [comment]
You can't import <script> tags into page contents. You need to use the page scripts (under the Page Properties dialog) and initialize it as shown in https://docs.requarks.io/en/troubleshooting#how-to-execute-javascript-code-on-page-load

**Reply by lawrenceching** [reply]
Hi @NGPixel ,

Great thank for your reply.

I don't know much about HTML and JS, could u help to explain a little bit more about this?

I successfully import the asciinema player using the SCRIPTS tab:
<img width="1004" alt="image" src="https://github.com/requarks/wiki/assets/12114025/d32effb8-5c04-45ae-95c1-2b9506f33445">


but I can't do it in markdown content... which is expected. It's just a normal text in markdown document.
```
# Header

window.boot.register('page-ready', () => {
<img width="1004" alt="image" src="https://github.com/requarks/wiki/assets/12114025/d32effb8-5c04-45ae-95c1-2b9506f33445">
})

Your content here
```



And In the asciinema offical wiki, I saw that:

_The player shows up **right at the place where the script is pasted**. Let’s look at the following markup:_

_<p>This is some text.</p>_
_<script src="https://asciinema.org/a/14.js" id="asciicast-14" async></script>_
_<p>This is some other text.</p>_

Can "right at the place where the script is pasted" still work in window.boot.register()?
